### PR TITLE
Add dev_flash2+3

### PR DESCRIPTION
**Pull request by clienthax**

### CODE REVIEW

File: rpcs3/Emu/system_config.h [hunk 1]
```diff
@@ -96,6 +98,16 @@ struct cfg_root : cfg::node
...
 		}
 
+		std::string get_dev_flash2() const
+		{
+			return get(dev_flash2, "dev_flash2/");
+		}
+
+		std::string get_dev_flash3() const
+		{
+			return get(dev_flash3, "dev_flash3/");
```
Comment (elad335) [inline rpcs3/Emu/system_config.h:108]:
```suggestion
			return get(dev_flash3);
```

The need for this has been removed recently.

File: rpcs3/Emu/system_config.h [hunk 2]
```diff
@@ -96,6 +98,16 @@ struct cfg_root : cfg::node
 			return get(dev_flash);
 		}
 
+		std::string get_dev_flash2() const
+		{
+			return get(dev_flash2, "dev_flash2/");
```
Comment (elad335) [inline rpcs3/Emu/system_config.h:103]:
```suggestion
			return get(dev_flash2);
```

**Review by elad335 — CHANGES_REQUESTED** [review]

File: rpcs3/Emu/system_config.h (hunk 1 above)
Comment (clienthax) [reply rpcs3/Emu/system_config.h]: Done

File: rpcs3/Emu/system_config.h (hunk 2 above)
Comment (clienthax) [reply rpcs3/Emu/system_config.h]: Done

**Pull request merged**